cluster/examples/kubernetes/nfs/common.yaml

```diff
@@ -0,0 +1,160 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name:  rook-nfs-system
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: nfsservers.nfs.rook.io
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.state
+    description: NFS Server instance state
+    name: State
+    type: string
+  group: nfs.rook.io
+  names:
+    kind: NFSServer
+    listKind: NFSServerList
+    plural: nfsservers
+    singular: nfsserver
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
```

Comment:
similarly, was this open API v3 schema created by hand, or generated with kubebuilder directives?

Reply:
answered in this conversation https://github.com/rook/rook/pull/5650#discussion_r444793528